packages/design-tokens/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This file should be named `tsconfig.dist.json` to be consistent with other package's dist config.